### ISSUE TITLE
Aquarius failing on rinkeby

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/aquarius#351]
In the course of trying to solve what looked like an [ocean.py error](https://app.zenhub.com/workspaces/ocean-main-5e5e5a97e91ce03f380fb9b7/issues/oceanprotocol/ocean.py/89), it appears that Aquarius is acting up.

It seems aquarius is failing on rinkeby. Of the commands below, the first two work, and the third fails. 
```console
curl https://aquarius.mainnet.oceanprotocol.com/api/v1/aquarius/assets
curl https://aquarius.ropsten.oceanprotocol.com/api/v1/aquarius/assets
curl https://aquarius.rinkeby.oceanprotocol.com/api/v1/aquarius/assets
```

**Comment by trentmc** [comment]
Details on errors.

```console
curl https://aquarius.rinkeby.oceanprotocol.com 
{"plugin":"elasticsearch","software":"Aquarius","version":"2.1.12"}

curl https://aquarius.rinkeby.oceanprotocol.com/api/v1/aquarius/assets 
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"> 
<title>500 Internal Server Error</title> 
<h1>Internal Server Error</h1> 
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

**Comment by trentmc** [comment]
![image.png](https://images.zenhubusercontent.com/5df770bc5c7a2019891bc4a0/7e5cf785-1dcb-4953-9b8f-8ec3aa20a197)

**Comment by trentmc** [comment]
Update: Samer got this fixed on rinkeby, so that it works.

So we can close the ticket.

**Comment by trentmc** [comment]
Update 2: Alex learned more about the fundamental problem.

From Alex in slack:

looks like we have assets without ids in aquarius rinkeby

not sure why is that allowed, probably it was published from outside
(later edit:  this is not possible.  means that we have some records in es without 'id'.  not sure how we end up with that)

quick fix:  https://github.com/oceanprotocol/aquarius/pull/352

**Comment by trentmc** [comment]
Samer's comments to Alex's slack comments:

those were the last_block saved from the event monitor

events_last_block and last_block

**Comment by alexcos20** [comment]
closed in #352
